tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCheckCLDR.java

```diff
@@ -491,6 +495,28 @@ public void TestAllLocales() {
         logln("Count:\t" + locales.size());
     }
 
+    private void checkNullWithInheritanceMark(String locale) {
+        CLDRFile resolved = cldrFactory.make(locale, true);
+        CLDRFile unresolved = resolved.getUnresolved();
+        for (String path : unresolved.fullIterable()) {
+            String value = unresolved.getStringValue(path);
+            if (CldrUtility.INHERITANCE_MARKER.equals(value)) {
+                switch (NameType.fromPath(path)) {
+                    case LANGUAGE:
+                    case SCRIPT:
+                    case TERRITORY:
+                    case VARIANT:
+                        assertNotNull(locale + " " + path, resolved.getStringValue(path));
+                        break;
+                    default:
+                        if (!logKnownIssue("CLDR-18294", "Null with inheritance mark")) {
```

Comment:
needs to be a new ticket (48) for after 18294 is already fixed.

Reply:
I'll follow up with that; merging now to fix the data